tests/integration/ha_tests/test_ha.py

```diff
@@ -140,6 +140,7 @@ async def test_scale_up_capablities(ops_test: OpsTest, continuous_writes) -> Non
     await verify_writes(ops_test)
 
 
+@pytest.mark.usefixtures("only_without_juju_secrets")
```

Comment:
As indicated in the corresponding, specific commit: this is a temporary change until https://github.com/canonical/mongodb-k8s-operator/pull/176 is merged. I'll rebase on that, wanna have relevant pipelines until.